scipy/stats/_stats_py.py

```diff
@@ -1680,14 +1685,19 @@ def kurtosistest(a, axis=0, nan_policy='propagate', alternative='two-sided'):
     pvalue : float
         The p-value for the hypothesis test.
 
-    See Also
-    --------
-    :ref:`hypothesis_kurtosistest` : Extended example
+    Raises
+    ------
+    ValueError
+        If shape of `a` along the given `axis` is less than 5.
 
     Notes
     -----
     Valid only for n>20. This function uses the method described in [1]_.
 
+    See Also
```

Comment:
See Also comes before Notes, as it was before.
https://numpydoc.readthedocs.io/en/latest/format.html#see-also